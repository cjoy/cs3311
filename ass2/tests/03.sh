# Pattern-based subject group: TELE412[01]
php members 1490
