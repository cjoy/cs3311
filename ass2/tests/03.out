TELE4120
TELE4121
